sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/guava25/annotations/Beta.java

```diff
@@ -0,0 +1,33 @@
+package com.azure.cosmos.implementation.guava25.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
```

Comment:
I think when we shade source code like this one from Guava we need to follow wacertain review process with legal/CELA - and also clearly call out the annotations for this external OSS ebing used (and make sure licensing information reflect it).

I think Kushagra might have more context - Mo had to do it in the past - and I think when ANTLr was attempted to be added the same thing happened.

Reply:
Me and @kushagraThapar went through a similar exercise for shading the `commons-codec` library. I shade all my classes from `guava` v25.0. We already make a "citation" for Guava-based dependencies in `NOTICE.txt` and `cgmanifest.json`. Correct me if any further action is needed @kushagraThapar. cc: @FabianMeiswinkel